### PR TITLE
nmcli-general, nmcli-networking: add page

### LINKED ISSUE #6167: Let's document: nmcli-*
The command is very useful, running a simple command to edit wifi config is both more initiative and often more effective than using a GUI menu for NetworkManager

These pages are poorly documented 

- [ ] general → #8264 
- [ ] networking → #8264
- [x] radio
- [x] connection → #6185 , (don't tick this)
- [x] device
- [x] agent
- [x] monitor → #6168 
- [x] nmcli (forwarding page edit) → #6203 

I haven't checked through these commands however some need a review and some need to be wrote entirely

### PR DESCRIPTION
**Pull request by marchersimon**
nmcli tool, version 1.38.2-2

Closes #6167

### CODE REVIEW

**Review by CleanMachine1 — APPROVED** [review]

File: pages/linux/nmcli-general.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# nmcli general
+
+> Manage general settings of NetworkManager.
+> This subcommand can also be called with `nmcli g`.
+> More information: <https://networkmanager.dev/docs/api/latest/nmcli.html>.
+
+- Show the general status of NetworkManager:
```
Comment (EmilySeville7cfg) [inline pages/linux/nmcli-general.md:7]:
```suggestion
- Show a general status of NetworkManager:
```

File: pages/linux/nmcli-general.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+`nmcli general`
+
+- Show the hostname of the current device:
```
Comment (EmilySeville7cfg) [inline pages/linux/nmcli-general.md:11]:
```suggestion
- Show a hostname of the current device:
```

File: pages/linux/nmcli-general.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+`nmcli general hostname`
+
+- Change the hostname of the current device:
```
Comment (EmilySeville7cfg) [inline pages/linux/nmcli-general.md:15]:
```suggestion
- Change a hostname of the current device:
```

File: pages/linux/nmcli-general.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+`sudo nmcli general hostname {{new_hostname}}`
+
+- Show the permissions of NetworkManager:
```
Comment (EmilySeville7cfg) [inline pages/linux/nmcli-general.md:19]:
```suggestion
- Show permissions of NetworkManager:
```

File: pages/linux/nmcli-general.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+`nmcli general permissions`
+
+- Show the current logging level and domains:
+
+`nmcli general logging`
+
+- Set the logging level and/or domains (see `man NetworkManager.conf` for all available domains):
```
Comment (EmilySeville7cfg) [inline pages/linux/nmcli-general.md:27]:
```suggestion
- Set a logging level and/or domains (see `man NetworkManager.conf` for all available domains):
```

File: pages/linux/nmcli-networking.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# nmcli networking
+
+> Manage the networking status of NetworkManager.
+> This subcommand can also be called with `nmcli n`.
+> More information: <https://networkmanager.dev/docs/api/latest/nmcli.html>.
+
+- Show the networking status of NetworkManager:
```
Comment (EmilySeville7cfg) [inline pages/linux/nmcli-networking.md:7]:
```suggestion
- Show a networking status of NetworkManager:
```

File: pages/linux/nmcli-networking.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+`nmcli networking`
+
+- Enable or disable networking and all interfaces managed by NetworkManager:
+
+`nmcli networking {{on|off}}`
+
+- Show the last known connectivity state:
```
Comment (EmilySeville7cfg) [inline pages/linux/nmcli-networking.md:15]:
```suggestion
- Show a last known connectivity state:
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
LGTM, just several suggestions. ;)

File: pages/linux/nmcli-general.md (hunk 1 above)
Comment (marchersimon) [reply pages/linux/nmcli-general.md:7]: But there isn't `a general status`, there's just `the general status`. There's only one single status, so we should use the definite article `the`.

File: pages/linux/nmcli-general.md (hunk 4 above)
Comment (marchersimon) [reply pages/linux/nmcli-general.md:19]: Why should we remove `the`? It's a lot less formal without.

File: pages/linux/nmcli-general.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages/linux/nmcli-general.md:7]: Yes, it single, but does it matter what status it is? For user maybe, but for us no. :)

Comment (CleanMachine1) [reply pages/linux/nmcli-general.md:7]: `the` is correct here

File: pages/linux/nmcli-general.md (hunk 3 above)
Comment (CleanMachine1) [reply pages/linux/nmcli-general.md:15]: `a` indicates that there is more than 1 hostname

File: pages/linux/nmcli-general.md (hunk 4 above)
Comment (EmilySeville7cfg) [reply pages/linux/nmcli-general.md:19]: Should we write formal tldr pages or not?

File: pages/linux/nmcli-general.md (hunk 3 above)
Comment (EmilySeville7cfg) [reply pages/linux/nmcli-general.md:15]: Thanks! I always thought that `a` just used when it doesn't matter what we are referring to but it seems it also used when there are more than one object in some context. In the context of some PC there is just one hostname. Now I hopefully understand why `the` is used here.

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! ❤️

**Pull request merged**